### ISSUE TITLE
Replace route from right component

### ISSUE THREAD

**Issue by fabianbormann** [react-native-simple-router-community/react-native-simple-router#65]
Hi, 

is it possible to replace a route from the right component?
I want to use a "Logout" button component which redirects to the Login screen after  
unset the credentials. But I get a Exception if I try to use this.props.resetToRoute.
this.props.toRoute works .. But I dont't want to see a "back" button.

```
Unhandled JS Exception: this.props.resetToRoute is not a function. (In 'this.props.resetToRoute({
name:'Login',
component:_Login2.default})'
```

Thank you!

**Comment by charpeni** [comment]
Hi @fabianbormann,

At this time, the only available function inside `leftCorner` and `rightCorner` is `toRoute()`. I will fix this soon.

Until then, you can pass the other functions with `rightCornerProps` along with `rightCorner` : 

``` javascript
rightCornerProps: {
  resetToRoute: this.props.resetToRoute,
},
```

**Comment by davidLeonardi** [comment]
Fixed by #68
